gpflow/models/gpr.py

```diff
@@ -36,18 +36,19 @@ class GPR(GPModel):
     and is given by
 
     .. math::
-       \\log p(\\mathbf y \\,|\\, \\mathbf f) =
-            \\mathcal N\\left(\\mathbf y\,|\, 0, \\mathbf K + \\sigma_n \\mathbf I\\right)
+       \log p(\mathbf y \,|\, \mathbf f) =
+            \mathcal N\left(\mathbf y\,|\, 0, \mathbf K + \sigma_n \mathbf I\right)
     """
 
-    def __init__(self, data: Data, kernel: Kernel, mean_function: Optional[MeanFunction] = None):
-        likelihood = gpflow.likelihoods.Gaussian()
+    def __init__(self, data: Data, kernel: Kernel, mean_function: Optional[MeanFunction] = None,
+                 noise_variance: int = 1.0):
```

Comment:
shouldnt this be a `float`, not `int`.
```suggestion
                 noise_variance: float = 1.0):
```